src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/X509Certificates/UnixPkcs12Reader.cs

```diff
@@ -139,7 +134,7 @@ public void Dispose()
 
                     fixed (byte* ptr = tmp)
```

Comment:
Should we just expose the pointer directly?  It doesn't really matter; it's just a bit of a headscratcher seeing this code pin a span and then free the pointer out from under it.

Reply:
I believe this is the only place that combines the Pointer Memory Manager and an alloc/free; so it seems "better" to me to just have a weird line of code here than to de-encapsulate the pointer.

Reply:
And/or use `Unsafe.AsPointer` to get the pointer instead of pinning:

`Unsafe.AsPointer(ref MemoryMarshal.GetReference(tmp))`